test/unit/vl-infoblock.test.html

```diff
@@ -44,13 +44,20 @@
       test('de titel is ingevuld als het titel attribuut wordt ingevuld', () => {
         const infoblock = fixture('vl-infoblock-fixture');
         
-        let title = infoblock.shadowRoot.querySelector("#infoblock_title");
-        expect(title.textContent).to.be.empty;
+        let title = infoblock.shadowRoot.querySelector("#infoblock_title slot").assignedElements();
+        expect(title).to.be.empty;
         
         const titel = 'Dit is een titel';
```

Comment:
Misschien beter hernoemen naar `text`, nu is er een title en titel variable, wat nogal verwarrend is.